met-web/src/components/engagement/admin/create/authoring/AuthoringSummary.tsx

```diff
@@ -83,62 +113,106 @@ const AuthoringBanner = () => {
+                                    sx={{ backgroundColor: colors.surface.white }}
+                                    id="title"
+                                    value={field.value}
+                                    error={undefined}
+                                    counter
+                                    maxLength={60}
+                                    placeholder="Section heading text"
+                                    onChange={(value) => {
+                                        field.onChange(handleTitleChange(value));
+                                    }}
+                                />
+                            );
+                        }}
                     />
                 </label>
             </Grid>
 
             <Grid sx={{ ...formItemContainerStyles, backgroundColor: colors.surface.blue[10] }} item>
-                <label htmlFor="body_copy">
+                <label htmlFor="editor_state">
                     <MetBigLabel style={metBigLabelStyles} role="document" tab-index="0">
                         Body Copy
                         <span style={{ fontWeight: 'normal' }}> (Required)</span>
                     </MetBigLabel>
                     <FormDescriptionText style={formDescriptionTextStyles}>
                         Your section heading should be descriptive, short and succinct.
                     </FormDescriptionText>
-                    <RichTextArea
-                        ariaLabel="Body Copy: Your section heading should be descriptive, short and succinct."
-                        spellCheck
-                        editorState={editorState}
-                        onEditorStateChange={handleEditorChange}
-                        handlePastedText={() => false}
-                        editorStyle={{
-                            height: '15em',
-                            padding: '1em',
-                            resize: 'vertical',
-                            backgroundColor: colors.surface.white,
-                        }}
-                        toolbar={toolbar}
-                    />
+                    <Suspense>
+                        <Await resolve={content}>
+                            {(content) => (
+                                <Controller
+                                    control={control}
+                                    name="editor_state"
+                                    rules={{ required: true }}
+                                    render={({ field }) => {
+                                        return (
+                                            <RichTextArea
+                                                ariaLabel="Body Copy: Your section heading should be descriptive, short and succinct."
+                                                spellCheck
+                                                initialContentState={
+                                                    tryParse(content[0].json_content)
```

Comment:
Actually I'm taking a second look at this and it's not quite the same. The first function just checks if the JSON is valid, the second one actually parses it. I need both unfortunately.

Reply:
But `tryParse` returns parsed JSON too, doesn't it?

Reply:
...and upon third inspection, I think this actually will work, since I am returning the value if it is parseable (I thought I was just returning true).